packages/grafana-ui/src/components/Table/utils.ts

```diff
@@ -123,6 +125,13 @@ export function getColumns(
       }
     };
 
+    const cellOptions = getCellOptions(field);
+    if (shouldRecalculateScaleRange(cellOptions, field)) {
+      field.state!.range = getMinMaxAndDelta(field);
+      field.config.min = field.state!.range.min;
+      field.config.max = field.state!.range.max;
+    }
```

Comment:
this feels a bit hacky, overwriting the logic in applyFieldOverrides. I would have thought the change / option check would be here:
https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/field/fieldOverrides.ts#L171  

that way it will work for all cells types (incl chart cell)